pythia/modules/losses.py

```diff
@@ -160,7 +160,7 @@ def forward(self, sample_list, model_output, *args, **kwargs):
         if loss.dim() == 0:
             loss = loss.view(1)
 
-            key = "{}/{}/{}".format(
```

Comment:
Was this a bug earlier?

Reply:
Yeah, I noticed this and fixed it in the rebase. It was already fixed in dev branch,